codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/message/MessageMarshallableGenerator.kt

```diff
@@ -21,117 +22,76 @@ class MessageMarshallableGenerator(
     private val payloadContentType: String
 ) {
     internal fun render(streamShape: UnionShape) {
```

Comment:
This generator takes what was the XML implementation, and makes it universal for all protocols.